doc/geany.txt

```diff
@@ -1689,12 +1689,13 @@ Global tags files can have three different formats:
 * Pipe-separated format
 * CTags format
 
-The first line of global tags files should be a comment, introduced
-by ``#`` followed by a space and a string like ``format=pipe``,
-``format=ctags`` or ``format=tagmanager`` respectively, these are
+For the tagmanager or pipe-separated format, the first line of global tag files
+should be a comment, introduced by ``#`` followed by a space and a string like
+``format=tagmanager`` or ``format=pipe``, respectively; these are
 case-sensitive.  This helps Geany to read the file properly. If this
 line is missing, Geany tries to auto-detect the used format but this
-might fail.
+might fail.  Tag files using the CTags format should be left unmodified in the
```

Comment:
"used format" above s/b "format used"

(I know its not your change, but might as well fix it)

Reply:
OK.